samples/ChatSample/Views/Home/Index.cshtml

```diff
@@ -34,6 +64,14 @@ document.getElementById('sendmessage').addEventListener('submit', event => {
         document.getElementById('messages').appendChild(child);
     });
 
+    newMessage.value = ''; 
     event.preventDefault();
 });
+
+let addUserOnline = function (user) {
```

Comment:
It was introduced by the person who originally submitted the PR and I did not notice it. I don't think it needs to be a variable at all.